server/models/ride.ts

```diff
@@ -27,6 +27,11 @@ export type RideType = {
   endTime: string,
   rider: RiderType,
   driver?: DriverType,
+  recurring?: boolean,
+  recurringDays?: string[],
+  endDate?: string
+  deleted?: boolean,
```

Comment:
Maybe think of a better name than `deleted` as this can indicate that a ride has been independently edited as well, according to you design document.

Reply:
If a ride is independently edited, 2 new one-time rides are created (1 is deleted indicating the instance that shouldn't be scheduled, and the other is the new modified one that is replacing that instance)

Reply:
Why still store the delete one?

Reply:
So that we know not to schedule that instance of the recurring ride. So if a ride that is normally Monday at 12 is rescheduled to Tuesday at 12 for 1 week, we have to note the Monday ride that has been rescheduled without deleting the entire recurring ride

Reply:
So this would be needed if a ride is rescheduled before it has been scheduled. If it has already been scheduled then we don't have to store the deleted instance separately